test/elm/clinical/clinical-test.ts

```diff
@@ -500,4 +500,53 @@ describe('CalculateAgeAt', () => {
     setup(this, data, [p3]);
     (await this.calculateAgeInYearsDateArg.exec(this.ctx)).should.eql(new Uncertainty(17, 18));
   });
+
+  it('should convert date to DateTime, give 18 since the timeZoneOffset is 0', async function () {
+    setup(this, data, [p6]);
+    // Execute these tests as if it is 2024-01-01 at 00:00:00.0000 GMT
+    this.ctx.executionDateTime = new DT.DateTime(2024, 1, 1, 0, 0, 0, 0, 0);
+    // Change it to the Date class
+    this.ctx.patient.birthDate = new DT.Date(2005, 12, 31);
+
+    (await this.ageAtJanuary1DateTimeArg.exec(this.ctx)).should.equal(18);
+  });
+
+  it('should convert date to DateTime, give 18 since the timeZoneOffset on the parent is 0', async function () {
+    setup(this, data, [p6]);
+    this.ctx.executionDateTime = undefined;
+    // Execute these tests as if it is 2024-01-01 at 00:00:00.0000 GMT
+    this.ctx.parent.getExecutionDateTime = () => {
+      return new DT.DateTime(2024, 1, 1, 0, 0, 0, 0, 0);
+    };
+    // Change it to the Date class
+    this.ctx.patient.birthDate = new DT.Date(2005, 12, 31);
+
+    (await this.ageAtJanuary1DateTimeArg.exec(this.ctx)).should.equal(18);
+  });
+
+  it('should convert date to DateTime, but give an Uncertainty since the timeZoneOffset is not passed in and therefore equal to the timezone of the evaluation request timestamp', async function () {
```

Comment:
Thanks for the comments, @cmoesel ! The only one I am unsure about is this one. Maybe I did things incorrectly here and/or I am interpreting things incorrectly (dateTimes still make me crazy 😄 ) but I believe that since this is testing `AgeInYearsAt`- it is getting `timeZoneOffset` from `CalculateAgeAt` (`const timeZoneOffset = ctx.getExecutionDateTime().timezoneOffset;`. It's possible that I did that wrong, but I think the timezoneOffset is that is being passed is -4 (the timezone of the evaluation request timestamp).

Reply:
Oh, right! You are correct. That was my mistake. Thanks for pointing me to the code where that happens. Sorry about the false alarm!

Reply:
No worries! I wasn't totally sure either so thank you!